### PR TITLE
 Fix: update valine comment system version and update configuration

### PR DESCRIPTION
**Pull request by xCss**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [ ] The commit message follows [our guidelines](https://github.com/iissnan/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [ ] Tests for the changes have been added (for bug fixes / features).
   - [ ] Muse | Mist have been tested.
   - [ ] Pisces | Gemini have been tested.
- [x] Docs have been added / updated (for bug fixes / features).

## PR Type
**What kind of change does this PR introduce?**  

- [x] Bugfix.
- [ ] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number(s): N/A

## What is the new behavior?
Description about this pull, in several words...

* Screens with this changes: N/A
* Link to demo site with this changes: N/A

### How to use?
In NexT `_config.yml`:
```yml
-# more info please open https://github.com/xCss/Valine
+# more info please open https://valine.js.org
 valine:
   enable: false
   appid:  # your leancloud application appid
   appkey:  # your leancloud application appkey
   notify: false # mail notifier , https://github.com/xCss/Valine/wiki
   verify: false # Verification code
-  placeholder: Comment input placeholder
+  placeholder: Just go go # comment box placeholder
+  avatar: mm # gravatar style
+  guest_info: nick,mail,link # custom comment header
+  pageSize: 10 # pagination size
```
`layout/_macro/post.swig`:
```swig
+            {% elseif theme.valine.enable and theme.valine.appid and theme.valine.appkey %}
+              <span class="post-comments-count">
+                <span class="post-meta-divider">|</span>
+                <span class="post-meta-item-icon">
+                  <i class="fa fa-comment-o"></i>
+                </span>
+                <a href="{{ url_for(post.path) }}#comments" itemprop="discussionUrl">
+                  <span class="post-comments-count valine-comment-count" data-xid="{{ url_for(post.path) }}" itemprop="commentCount"></span>
+                </a>
+              </span>
```

`layout/_third-party/comments/valine.swig`
```swig
-{% if page.comments and theme.valine.enable and theme.valine.appid and theme.valine.appkey %}
+<script src="//cdn1.lncld.net/static/js/3.0.4/av-min.js"></script>
+<script src="//unpkg.com/valine/dist/Valine.min.js"></script>
+{% if theme.valine.enable and theme.valine.appid and theme.valine.appkey %}
   
-  <script src="//cdn1.lncld.net/static/js/3.0.4/av-min.js"></script>
-  <script src="//cdn.jsdelivr.net/gh/xcss/valine@v1.1.7-rc3/dist/Valine.min.js"></script>
   <script type="text/javascript">
+    var GUEST = ['nick','mail','link'];
+    var guest = '{{ theme.valine.guest_info }}';
+    guest = guest.split(',').filter(item=>{
+      return GUEST.indexOf(item)>-1;
+    });
     new Valine({
-        av: AV,
         el: '#comments' ,
         verify: {{ theme.valine.verify }},
         notify: {{ theme.valine.notify }},
-        app_id: '{{ theme.valine.appid }}',
-        app_key: '{{ theme.valine.appkey }}',
-        placeholder: '{{ theme.valine.placeholder }}'
+        appId: '{{ theme.valine.appid }}',
+        appKey: '{{ theme.valine.appkey }}',
+        placeholder: '{{ theme.valine.placeholder }}',
+        avatar:'{{ theme.valine.avatar }}',
+        guest_info:guest,
+        pageSize:'{{ theme.valine.pageSize }}' || 10,
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

File: layout/_partials/comments.swig [hunk 1]
```diff
@@ -64,7 +64,6 @@
 
   {% elseif theme.valine.appid and theme.valine.appkey %}
     <div class="comments" id="comments">
-      <div id="vcomments"></div>
```
Comment (ivan-nginx) [inline layout/_partials/comments.swig]: No need apeear comments block on full post here?

Comment (xCss) [reply layout/_partials/comments.swig]: Ok, i got it. thx

**Comment by tingstory** [comment]
是 Markdown 不是 MarkDown

**Comment by tingstory** [comment]
Error：MarkDown is Support
is Markdown not MarkDown

**Comment by xushankun** [comment]
### _你好啊**你好我很好**_

**Comment by iloveuav** [comment]
我可以显示评论块  一切正常  就是点击评论后  没有任何反应，learnCloud也没反应，appid那些都配置好了的  没有任何报错提示~    网站是  https://www.uavserve.online/

**Comment by iloveuav** [comment]
我可以显示评论块 一切正常 就是点击评论后 没有任何反应，learnCloud也没反应，appid那些都配置好了的 没有任何报错提示~ 网站是 https://www.uavserve.online/         救救孩子吧~

**Comment by iloveuav** [comment]
> > 我可以显示评论块 一切正常 就是点击评论后 没有任何反应，learnCloud也没反应，appid那些都配置好了的 没有任何报错提示~ 网站是 https://www.uavserve.online/ 救救孩子吧~
> 
> image
> 
> 只能帮你到这了，加油

太感谢了  就是#号前面没有空格导致的   没想到几乎秒回  太感动了

**Pull request merged**